### DISCUSSION TITLE
Can I set border rounded corners in windows Platform mode?

### DISCUSSION THREAD

**Discussion by wj8400684** [amwx/FluentAvalonia#267] [Q&A]
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/39239954/211131310-ab65178c-b9a3-4944-acdc-dec855b226a6.png">

**Comment by amwx** [comment]
I'm not sure I understand your question. Could you elaborate?

**Reply by wj8400684** [reply]
<img width="1247" alt="image" src="https://user-images.githubusercontent.com/39239954/211256610-81f7dd0f-b2a5-4d83-8d70-79e05eed82ae.png">
Border CornerRadius!
